integration/build_env.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
Should this file be combined with build_func.sh?

Reply:
If this was just going to be used for building the apps I'd agree.

I was hoping to use this to get the compiler wrappers set for the GEOPM build as well.  The other code in ```build_func.sh``` doesn't make much sense in that case (unless we're going to put a func together to fo the GEOPM build too...).